qemu/tests/qemu_guest_agent.py

```diff
@@ -3150,13 +3150,12 @@ def log_check(qga_cmd):
             """
             error_context.context("Check %s cmd in agent log." % qga_cmd,
                                   LOG_JOB.info)
+            get_log_cmd = params["get_log_cmd"] % qga_cmd
             log_str = session.cmd_output(get_log_cmd).strip().split('\n')[-1]
-            pattern = r"%s" % qga_cmd
-            if not re.findall(pattern, log_str, re.M | re.I):
+            if not log_str:
```

Comment:
This would make the checkpoint meaningless, I would suggest to check at least the command return value if we don't check the detail content.